modules/dojo.py

```diff
@@ -33,14 +33,16 @@ def create_room(self, room_name, room_type):
                 return room
 
         if room_type.lower() == 'office':
-            new_office = officeSpace(room_name)
+            room_id = len(self.all_rooms) +1
+            new_office = officeSpace(room_id, room_name)
             self.all_rooms.append(new_office)
             self.all_offices.append(new_office)
             self.avialable_offices.append(new_office)
             return new_office
```

Comment:
I see you are returning objects, you should return a more elaborate return message such as 'office '+room_name+' was successfully created.' and if the room was not able to be created, have a return statement that gives that message.